src/python/twitter/common/reviewboard/reviewboard.py

```diff
@@ -103,7 +103,7 @@ def _add_auth_params(self, username, password):
     urllib2.install_opener(opener)
 
   def get_url(self, rb_id):
-    return '%s/r/%s' % (self.url, rb_id)
+    return '%sr/%s' % (self.url, rb_id)
```

Comment:
I've changed it to use urljoin, which will produce the correct result regardless of whether self.url has a '/'.